test/scripts/create_all_hpp.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+# -----------------------------------------------------------------------------------------------------
+# Copyright (c) 2006-2021, Knut Reinert & Freie Universität Berlin
+# Copyright (c) 2016-2021, Knut Reinert & MPI für molekulare Genetik
+# This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+# shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+# -----------------------------------------------------------------------------------------------------
+#
+# Usage: create_all_hpp.sh <SeqAn3 root directory>
+# Will create all.hpp in all subdirectories
+
+# Directories (+ subdirectories) to ignore (not creating any all.hpp files in these folders)
+exceptionList=(
+    include/seqan3/std
+    include/seqan3/contrib
+    /detail
+    /exposition_only
+    include/seqan3/core/concept
+    include/seqan3/utility/concept
+)
+
+# prints the header of a typical all.hpp file
+print_header()
+{
+cat << EOF
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2021, Knut Reinert & Freie Universität Berlin
```

Comment:
Since everyhting else is done. this can be a follow up (if done at all)

Reply:
Good idea, of course we can make it the current year, this way this script doesn't have to be updated  :-)

I just want to point out, that we have also `update_copyright.sh` which should be used to actually update the copyright year, since it will change it in all files and not just all.hpp files.

Reply:
ah you are right. then I think we do not need it